### PR TITLE
Mute logs in tests

### PR DESCRIPTION
**Pull request by robert-zaremba**
Files to watch:
* examples/*/package.json
* packages/*/.mocharc.json

### CODE REVIEW

File: packages/algob/test/internal/core/config/config-loading.ts [hunk 1]
```diff
@@ -205,14 +205,15 @@ describe("config loading", function () {
     });
 
     it("should detect conflict of account names", async function () {
-      const spy = sinon.spy(console, 'warn');
+      // console is mocked in package.json mocha options
+      let stub = console.warn as sinon.SinonStub;
```
Comment (ratik21) [inline packages/algob/test/internal/core/config/config-loading.ts:209]: curious - what's the difference between using `spy` and `stub`? Is using `stub` better ?

File: test/setup.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+beforeEach(() => {
+  for (let l of levels)
+    console[l].reset();
+})
+
+
+after(() => {
+  for (let l of levels)
+    console[l].restore();
+})
```
Comment (ratik21) [inline test/setup.js:20]: could you  please add a doc string (maybe at the top of file), regarding suppressing console logs in setup?

**Review by ratik21 — APPROVED** [review]
left one comment

File: packages/algob/test/internal/core/config/config-loading.ts (hunk 1 above)
Comment (robert-zaremba) [reply packages/algob/test/internal/core/config/config-loading.ts:209]: I was usually using stubs, but it seams it's better to use spy here. I will update to spy. Thanks for asking.
here is an overview: https://www.javatpoint.com/mock-vs-stub-vs-spy

File: test/setup.js (hunk 2 above)
Comment (robert-zaremba) [reply test/setup.js:20]: yes ,good idea.

**Pull request merged**